transport/nio/pom.xml

```diff
@@ -61,6 +61,61 @@
             <artifactId>jmock-legacy</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <!-- Needed for EL only, not AST -->
+            <groupId>org.kaazing</groupId>
+            <artifactId>k3po.lang</artifactId>
+            <version>${project.version}</version>
```

Comment:
This version setting is causing build failures because develop-SNAPSHOT version of k3po is not available to Jenkins builds. Please use ${k3po.version} instead (which will result in version 2.0.0 being used, see root pom.xml for gateway)